client/src/components/modals/ConfirmDeleteModal.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Modal, ModalHeader, ModalBody, ModalFooter, Button } from 'reactstrap';
+
+const ConfirmDeleteModal = ({ isOpen, toggle, confirmDelete }) => {
+  return (
+    <Modal isOpen={isOpen} toggle={toggle}>
+      <ModalHeader toggle={toggle}>Confirm Deletion</ModalHeader>
+      <ModalBody>
+        Are you sure you want to delete this category?
```

Comment:
Instead of hardcoding the text "category" on this line, it should be a string referencing a passed in prop, so that this modal can be reused for things other than categories